frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -187,10 +257,10 @@ class SaveTripModal extends React.Component {
         .doc(tripId)
         .set(tripData)
         .then(() => {
-          console.log("Document written with ID: ", tripId);
+          console.log('Document written with ID: ', tripId);
```

Comment:
do you intend to keep these in the successful case? (same question)

Reply:
Same as above.